python-sdk/src/transfers/constants.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
```

Comment:
@sunank200 will the UniversalTransferOperator be shipped as an independent package or as part of the `python-sdk`?
If it is part of the same package, it may be confusing to have a separate constants file - and a different `FileLocation` definition than the rest.

Reply:
That was exactly the question I had here